src/fondant/pipeline/lightweight_component.py

```diff
@@ -1,14 +1,24 @@
+import inspect
+import itertools
+import textwrap
 import typing as t
 from dataclasses import dataclass
 from functools import wraps
 
+from fondant.component import Component
+
 
 @dataclass
 class Image:
     base_image: str = "fondant:latest"
     extra_requires: t.Optional[t.List[str]] = None
     script: t.Optional[str] = None
 
+    def __post_init__(self):
+        if self.base_image is None:
+            # TODO: link to Fondant version
```

Comment:
Is this TODO already somewhere noted down in an issue?

Reply:
It should be part of #782